CHANGELOG.md

```diff
@@ -1,18 +1,30 @@
-[//]: # "START/LATEST"
-
+[//]: # (START/LATEST)
 # Latest
 
 ## Features
-
-- A user-friendly description of a new feature. {issue-number}
+  * A user-friendly description of a new feature. {issue-number}
 
 ## Fixes
-
-- A user-friendly description of a fix. {issue-number}
+ * A user-friendly description of a fix. {issue-number}
 
 ## Security
+ * A user-friendly description of a security fix. {issue-number}
+
+---
+
+[//]: # "START/v1.3.0"
+
+# v1.3.0
+
+## Features
+
+- The TOTP code of a TOTP field can now be accessed using the `.totp` property of a field. {#33}
```

Comment:
We mention the field type to be `OTP` instead of `TOTP`.
```suggestion
- The TOTP code of a OTP field can now be accessed using the `.totp` property of a field. {#33}
```